VideoLocker/res/layout/fragment_course_unit_not_available.xml

```diff
@@ -0,0 +1,22 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="fill_parent"
+    android:layout_height="fill_parent"
+    android:background="@color/white"
+    android:gravity="center"
+    android:orientation="vertical">
+
+    <org.edx.mobile.view.custom.IconImageViewXml
+        android:layout_width="@dimen/large_indicator_icon_width"
+        android:layout_height="@dimen/large_indicator_icon_height"
```

Comment:
The width and height should be set to point to the `empty_list_icon_size` dimen resource. This could also be renamed to a more generic name, e.g. `content_unavailable_error_icon_size`. Similarly, the `empty_content_text` style could be renamed to `content_unavailable_error_text`, or something similar.